public/js/log.js

```diff
@@ -60,250 +59,295 @@ MAGI_log.resizes = {
 };
 MAGI_log.tooltips = [];
 
-$('#magi-loggingReadMore').click(function(e) {
-  e.preventDefault();
-  var isVisible = 'visible' == $('#magi-loggingReadMoreDiv').css('visibility'),
-      visibleState = isVisible ? 'hidden' : 'visible',
-      displayState = isVisible ? 'none' : 'block';
-
-  $('#magi-loggingReadMoreDiv').css('visibility', visibleState);
-  $('#magi-loggingReadMoreDiv').css('display', displayState);
-
+$().ready(function () {
+  $.get('/logEnabled', function(res, status) {
```

Comment:
Okay that sounds good, but I still think it makes more sense to pass a variable from the server. Then you don't even have to load the Javascript files, and it should be pretty easy to implement. I don't think you'd have to change any of log.js.